irispy/spectrograph.py

```diff
@@ -333,7 +333,8 @@ def convert_to(self, new_unit_type):
         if new_unit_type == "radiance" or self.unit.is_equivalent(iris_tools.RADIANCE_UNIT):
             # Get spectral dispersion per pixel.
             spectral_wcs_index = np.where(np.array(self.wcs.wcs.ctype) == "WAVE")[0][0]
-            spectral_dispersion_per_pixel = self.wcs.wcs.cdelt[spectral_wcs_index] * \ self.wcs.wcs.cunit[spectral_wcs_index]
+            spectral_dispersion_per_pixel = self.wcs.wcs.cdelt[spectral_wcs_index] *
```

Comment:
Removing the ```\``` AND breaking the line is *not* correct.  It is one of the other.  I suggest just adding a ```\``` like this:
```python
spectral_dispersion_per_pixel = self.wcs.wcs.cdelt[spectral_wcs_index] * \
   self.wcs.wcs.cunit[spectral_wcs_index]
```